code/modules/clothing/spacesuits/flightsuit.dm

```diff
@@ -102,8 +104,15 @@
 	var/obj/item/weapon/stock_parts/matter_bin/part_bin = null
 
 	var/crashing = FALSE	//Are we currently getting wrecked?
-	var/atom/movable/dragging_through = FALSE
+	var/atom/movable/cached_pull
+	var/turf/old_pull_turf
 
+/obj/item/device/flightpack/proc/changeWearer(mob/changeto)
+	if(wearer)
+		wearer.user_movement_hooks -= src
+	if(istype(changeto))
```

Comment:
@Cyberboss so I can just call changeWearer() to reset its wearer because if it's null it won't change.